python/cudf/cudf/tests/test_parquet.py

```diff
@@ -3768,6 +3788,39 @@ def test_parquet_chunked_reader(
     assert_eq(expected, actual)
 
 
+@pytest.mark.parametrize("chunk_read_limit", [240])  # [0, 240, 102400000])
+@pytest.mark.parametrize("pass_read_limit", [240])  # [0, 240, 102400000])
+@pytest.mark.parametrize(
+    "nrows,skip_rows",
+    [
+        # These are OK
+        (0, 0),
+        (1000, 0),
+        (0, 1000),
+        # segfault in the chunked parquet reader (invalid device ordinal)
+        (1000, 10000),
+    ],
+)
+def test_parquet_chunked_reader_nrows_skiprows(
+    chunk_read_limit, pass_read_limit, nrows, skip_rows
+):
+    df = pd.DataFrame(
+        {"a": [1, 2, 3, 4] * 100000, "b": ["av", "qw", "hi", "xyz"] * 100000}
+    )
+    buffer = BytesIO()
+    df.to_parquet(buffer)
+    actual = read_parquet_chunked(
+        [buffer],
+        chunk_read_limit=chunk_read_limit,
+        pass_read_limit=pass_read_limit,
+        nrows=nrows,
+        skip_rows=skip_rows,
+    )
+    buffer.seek(0)
+    expected = cudf.read_parquet(buffer, nrows=nrows, skip_rows=skip_rows)
+    assert_eq(expected, actual)
```

Comment:
```suggestion
def test_parquet_reader_nrows_skiprows(
    nrows, skip_rows
):
    df = pd.DataFrame(
        {"a": [1, 2, 3, 4] * 100000, "b": ["av", "qw", "hi", "xyz"] * 100000}
    )
    expected = df[skip_rows:skip_rows+nrows]
    buffer = BytesIO()
    df.to_parquet(buffer)
    got = cudf.read_parquet(buffer, nrows=nrows, skip_rows=skip_rows)
    # Check results
    assert_eq(expected, got)
```

Reply:
If you want, I can keep this as a skipped test for you instead if you need the extra test parametrizations on the Python side.

Reply:
No it's okay, we can remove it altogether for now or just test the normal reader. We can add some tests for the chunked reader when we have it working. Might be a few weeks.